assets/src/components/v2/departures/later_departures.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {
+  useEffect,
+  useLayoutEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import _ from "lodash";
+import cx from "classnames";
+import weakKey from "weak-key";
+
+import { hasOverflowX } from "Util/util";
+
+import DepartureRow from "./departure_row";
+import { Row } from "./normal_section";
+import RoutePill from "./route_pill";
+
+const MAX_LATER_DEPARTURES = 5;
+
+const LaterDepatures = ({ rows }: { rows: Row[] }) => {
+  const [limit, setLimit] = useState(MAX_LATER_DEPARTURES);
+  const departures = useMemo(
+    () =>
+      _.take(
+        rows.filter(
+          (r): r is DepartureRow & { type: "departure_row" } =>
+            r.type === "departure_row",
```

Comment:
Could we instead specify that the type of this component's prop (and of `trimmedRows`) is `DepartureRow[]`? Since ultimately we only ever trim / "later"-ify departure rows.